SingularityService/src/main/java/com/hubspot/singularity/data/StateManager.java

```diff
@@ -133,6 +134,10 @@ public void save(SingularityHostState hostState) throws InterruptedException {
       numTasks.incr(pendingTaskId.getRequestId());
     }
 
+    for (SingularityTaskId cleaningTaskId : taskManager.getCleanupTaskIds()) {
+      numTasks.decr(cleaningTaskId.getRequestId());
```

Comment:
I believe so. I'm imagining a request that should have 3 instances, but has 2 running and 1 cleaning. Since we're still counting pending tasks, if one task is pending, then the count is correct, otherwise, it's under provisioned. I think counting cleaning tasks may actually have made us miss potential under provisioned alerts in the past